src/util/api/likernft/book/cart.ts

```diff
@@ -0,0 +1,800 @@
+          shippingCost: shippingCostAmount,
+          originalPrice: originalPriceInDecimal / 100,
+          from,
+        }) : sendNFTBookPurchaseEmail({
+          email,
+          phone: phone || '',
+          shippingDetails: null,
+          shippingCost: 0,
+          originalPrice: originalPriceInDecimal / 100,
+          isGift: false,
+          giftInfo: null,
+          notificationEmails,
+          classId,
+          bookName,
+          priceName,
+          paymentId,
+          claimToken,
+          amountTotal: (amountTotal || 0) / 100,
+          mustClaimToView,
+          isPhysicalOnly: false,
+          from,
+        }),
+        sendNFTBookSalesSlackNotification({
+          classId,
+          bookName,
+          paymentId,
+          email,
+          priceName,
+          priceWithCurrency: `${convertedPriceInDecimal} ${convertedCurrency}`,
+          method: 'Fiat',
+          from,
+        }),
+        createAirtableBookSalesRecordFromStripePaymentIntent({
+          pi: capturedPaymentIntent,
+          classId,
+          collectionId,
+          priceIndex,
+          stripeFeeAmount,
+          stripeFeeCurrency,
+          from,
+          quantity,
+          feeInfo,
+          transfers,
+          shippingCountry: undefined,
+          shippingCost: undefined,
+        }),
+      ]);
+
+      publisher.publish(PUBSUB_TOPIC_MISC, req, {
+        logType: 'BookNFTPurchaseCaptured',
+        paymentId,
+        cartId,
+        fromChannel: from,
+        sessionId: session.id,
+        isGift: false,
+      });
+    }
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error(err);
+    const errorMessage = (err as Error).message;
+    const errorStack = (err as Error).stack;
+    publisher.publish(PUBSUB_TOPIC_MISC, req, {
+      logType: 'BookNFTPurchaseError',
+      type: 'stripe',
+      paymentId,
+      cartId,
+      error: (err as Error).toString(),
+      errorMessage,
+      errorStack,
+    });
+    await likeNFTBookCartCollection.doc(cartId).update({
+      status: 'canceled',
+      email,
+    });
+    await stripe.paymentIntents.cancel(paymentIntent as string)
+      .catch((error) => console.error(error)); // eslint-disable-line no-console
+  }
+}
+
+export async function handleNewCartStripeCheckout(items: CartItem[], {
+  gaClientId,
+  gaSessionId,
+  from: inputFrom,
+  email,
+  utm,
+}: {
+  gaClientId?: string,
+  gaSessionId?: string,
+  email?: string,
+  from?: string,
+  utm?: {
+    campaign?: string,
+    source?: string,
+    medium?: string,
+  },
+} = {}) {
+  const itemInfos: CartItemWithInfo[] = await Promise.all(items.map(async (item) => {
+    const {
+      classId,
+      priceIndex,
+      collectionId,
+      coupon,
+      customPriceInDecimal,
+      quantity = 1,
+      from: itemFrom,
+    } = item;
+    let info;
+    if (!Number.isInteger(quantity) || quantity <= 1) {
+      throw new ValidationError('QUANTITY_INVALID');
+    }
```

Comment:
```suggestion
    if (!Number.isInteger(quantity) || quantity < 1) {
      throw new ValidationError('QUANTITY_INVALID');
    }
```
If the quantity =1 , it will also throw an error